### DISCUSSION TITLE
临床医学模版

### DISCUSSION THREAD

**Discussion by xfz329** [TheNetAdmin/zjuthesis#409] [General]
作为使用本模版（最多的）用户之一，最近合作课题组的师妹毕业，帮她简单定制了下模板。目前也已通过学校盲审。
在想如果有必要，可以把临床医学的模板贡献下。
临床医学主要需要一个额外的医学综述，其他似乎都是细节调整。
但我目前只知道研究生的相关要求，不知道能否满足pr的条件。 @TheNetAdmin

**Comment by TheNetAdmin** [comment]
这个好呀！如果有空的话请务必提交一个pr，目前模板里还没有为医学院定制过模板，这次应该能让更多同学用到

**Reply by xfz329** [reply]
医学院的硕博论文模板结构上需要增加一个综述（相当于单独一章，但是不要这章下面所有的二级三级标题），可能和常规专业不一样，可能需要增加新的默认编译文件，我先实现个效果发个pdf上来看看，是否需要保留或增删?

**Reply by xfz329** [reply]
综述也需要单独于正文之外的参考文献

**Reply by gantanger111125** [reply]
> 医学院的硕博论文模板结构上需要增加一个综述（相当于单独一章，但是不要这章下面所有的二级三级标题），可能和常规专业不一样，可能需要增加新的默认编译文件，我先实现个效果发个pdf上来看看，是否需要保留或增删?

您好，请问模板做好了吗？正在写硕士毕业论文，很需要，万分感谢！

**Reply by xfz329** [reply]
今年初我已经基本完成了模板，我找空PR一下。目前已知的问题是目录似乎没有完美支持（如果我没记错的话）第二，我了解到的，临床医学相关论文中，会出现不少按第一部分，第二部分撰写内容的情况。我尝试按word的论文模式复现，发现复现有难度，我一直没完美解决。

**Reply by xfz329** [reply]
工作之后比较忙，最近在准备年底汇报。你可以先按正常模版写。不是我说的第二种情况可以等我pr下。第二种情况我只找到等同实现的方法，并没有完美解决。所以一直没有pr

**Reply by TheNetAdmin** [reply]
有空pr一下的话，我看看能不能搞定第一第二部分这个功能，我年底刚好有点空

**Reply by xfz329** [reply]
主要是我没解决自动生成目录问题，我找下图片

**Reply by xfz329** [reply]
我准备先拉一个临时的PR，医学院的研究生论文模板确实比较复杂，我之前的实现方式可能不太优雅，我们一起讨论下？ @TheNetAdmin

**Reply by TheNetAdmin** [reply]
好呀！

**Reply by xfz329** [reply]
@gantanger111125 你有没有医学院说明具体的模版文件啊？我之前是按我师妹提供的几篇论文手撕的

**Comment by Dragon-Wu** [comment]
> 作为使用本模版（最多的）用户之一，最近合作课题组的师妹毕业，帮她简单定制了下模板。目前也已通过学校盲审。 在想如果有必要，可以把临床医学的模板贡献下。 临床医学主要需要一个额外的医学综述，其他似乎都是细节调整。 但我目前只知道研究生的相关要求，不知道能否满足pr的条件。 @TheNetAdmin

你好，请问可以分享一下医学的模板吗？

**Reply by xfz329** [reply]
你需要写成分成两部分的那种吗？如果不需要，现在我做的可以比较方便实现。但是我今天在外出，可能要今天晚点或者明天才有空上传。

**Reply by Dragon-Wu** [reply]
你好！ 如果有两部分的最好了，不过合在的一起应该也行，我后面再想想能不能调整（或者再单独写一个拼起来?），可以麻烦先发我一份你现在做的吗？如果方便的话，可以发邮箱jiagengwu@zju.edu.cn或者网盘链接吗？感谢！

**Reply by xfz329** [reply]
今晚来不及了。我再赶下

**Comment by xfz329** [comment]
临时上传一个版本，为方便医学专业的同学直接撰写，这个版本与主版本相比，多了很多文件，同时有部分问题没有解决，故暂时没有直接PR @TheNetAdmin 。为方便大家交流，我打包了git记录。（如果会使用git的话，虽然这个对医学生苛刻了点）。但直接打包似乎超出了github的附件大小。贴一下百度网盘链接，zjuthesis_medical
链接: https://pan.baidu.com/s/1Mu-WgBmV7jrG3sjvEpY1kA?pwd=nva2 提取码: nva2 

不知道有没有更好的分享与临时协作方式？另外，排版效果可参见PDF文件。
[zjuthesis.pdf](https://github.com/user-attachments/files/19208353/zjuthesis.pdf)

**Reply by xfz329** [reply]
@Dragon-Wu

**Reply by xfz329** [reply]
上班之后太忙了，就不像在学校里时间那么多了。有问题可以先反馈看看

**Reply by xfz329** [reply]
已更新了一版

**Reply by Dragon-Wu** [reply]
太好了！ 麻烦同学了，感谢分享！

**Reply by xfz329** [reply]
你使用看看，有些细节我下班再回家处理下。公司电脑加密的比较麻烦

**Reply by xfz329** [reply]
你使用正常吗？有什么反馈意见吗 @Dragon-Wu

**Comment by JerryHuang0319** [comment]
百度网盘的zip下载后似乎说超过上传overleaf的大小了

**Reply by xfz329** [reply]
我上传的是完整的git仓库，如果上传过大，可以删除解压后文件夹中的.git 文件夹，以及out文件夹是编译中间临时文件的文件夹，也可以删除。再删除之后应该就不会超了，你试下

**Reply by JerryHuang0319** [reply]
非常感谢！我晚上回家尝试一下再来反馈！

**Reply by JerryHuang0319** [reply]
删除.git文件后，zip大小大幅缩减为600+kb，上传overleaf后可以打开但pdf部分显示错误，具体如图
<img width="1067" height="1235" alt="1" src="https://github.com/user-attachments/assets/dc7fad7a-54fa-4faa-8900-d73340fb6ac1" />

**Reply by xfz329** [reply]
我没有试过overleaf的在线编译，所以也没什么经验。或者本地试试？latexmk？

**Reply by JerryHuang0319** [reply]
很抱歉我刚好相反..就是我本身没有什么基础，所以朋友都推荐用overleaf来写会比较简单一点，我可能不太会部署本地环境这样

**Reply by xfz329** [reply]
工作之后非常忙，回复不怎么及时了。
我也不清楚这个overleaf到底是什么原因，
<img width="861" height="217" alt="image" src="https://github.com/user-attachments/assets/b8ea2da2-ca37-4767-a10b-ef3e216b7b00" />
这个能看懂，用了luatex，不可以，需要用xelatex 或者lunalatex.
本地配置也很简单的，一次配置就可以了。后面就一直命令行下敲个latexmk 就完事了

**Reply by JerryHuang0319** [reply]
没事没事，打扰到你了。我大概也清楚是要按introduction，实验方法实验结果讨论结论以及最后综述这样的顺序来写，我就直接在原模板里改一下好了，谢谢！

**Reply by xfz329** [reply]
没事没事，只是比较忙。有的时候顾不上回你。你要么试下本地编译，安装下latex，根据电脑性能大概30min-1h左右。然后latexmk就可以了。模板是可以用的。
有问题你还是可以问下的。

**Reply by JerryHuang0319** [reply]
谢谢！！刚好周末我在家里台式上先试一下看看！我人还在国外所以可能给你回复也有时差，忙的时候不用太care我的问题的：）

**Reply by xfz329** [reply]
好的，加油！祝顺利

**Reply by JerryHuang0319** [reply]
我想提问一个比较低级的问题，就是像大家放文献就是直接“找到对应文献-导出bibtex-复制粘贴到模板的ref.bib区域”这样吗？还是说会建一个类似zotero的库可以自动存之类的？

**Reply by xfz329** [reply]
我是这样处理的。不同的期刊导出的格式还是不一样的，我是说bibtex的字段。最后我还是统一处理了一下。

**Reply by xfz329** [reply]
已经可以编译了吗？

**Reply by JerryHuang0319** [reply]
理解了。就是正常需要的文献就导出bibtex然后粘到模板里然后引用，然后如果格式和标准的那种作者-期刊格式不一样就手动按一个模板调整一下就好。谢谢！另外就是我本地还是不太熟悉操作，所以还是没有办法在本地测试您的模板。我是根据您的模板在overleaf里作了调整（就是诸如去掉序言，加上综述，改一下章节title这样），真的很抱歉！

**Reply by xfz329** [reply]
OK 没事的。 overleaf上已经可以使用了是吧？感谢反馈！

**Reply by JerryHuang0319** [reply]
啊啊不好意思学长，没看到消息...不是在overleaf可以用了:(是我直接根据你提供的医学的模板在原有模板里自己修改...不过还是遇到了蛮多问题...比如说摘要和综述那里不知道怎么样加一个标题之类的...不好意思让学长误会了

**Reply by xfz329** [reply]
需要的话，你可以联系我。我可以帮你看看
